template.yaml

```diff
@@ -7,11 +7,95 @@ Parameters:
     Type: String
     Default: "/SungardAS/Alerts/Account"
     Description: Account Cloudwatch Log Group Name
-  SubscriptionFilterDestinationArn:
+  DestinationArn:
     Type: String
     Description: Subscription Filter Destination Arn
+  DbUserName:
+    Type: String
+    Description: user name of mcaws db
+  DbPassword:
+    Type: String
+    NoEcho: true
+    Description: password of mcaws db
+  DbHost:
+    Type: String
+    Description: Host of mcaws db
+  MasterMgmAwsId:
+    Type: String
+    Description: AWS Id of Master Management Account
```

Comment:
why is it mentioned as Master Management Account, it should "Management Account"